includes/User.php

```diff
@@ -23,6 +23,7 @@
 use Wikia\DependencyInjection\Injector;
 use Wikia\Logger\Loggable;
 use Wikia\Service\User\Preferences\UserPreferences;
+use Wikia\Service\User\Attributes\UserAttributes;
```

Comment:
Whoa! This is a huge pull request!

Fear not, fair code reviewer. The bulk of this is the auto-generated client code created by swagger-autogen. The code which I added to wire that client into MW starts [here](https://github.com/Wikia/app/pull/7943/files#diff-5f12f40345b8d749abbb52d78fda227eR1) which is much more manageable.